### DISCUSSION TITLE
Que se passe t-il exactement quand, on clique sur tout interdire ?

### DISCUSSION THREAD

**Discussion by Jjodar** [AmauriC/tarteaucitron.js#674] [Q&A]
Je n'ai pas trouvé de forum pour poser la question. 
Je souhaite savoir exactement ce qu'il se passe lorsque qu'on clique sur ton interdire. 
Par exemple, j'ai mon google Gtag active et qui s'affiche dans mon code source. Quand, je clique sur tout interdire, j'ai toujours mon Google Gtag qui est affiché de mon code source. 

merci d'avance. 
Jonathan

**Comment by AmauriC** [comment]
Il faut supprimer le Google tag du site et laisse tarteaucitron gérer le chargement ;-)
